tests/data/no_data_config.toml

```diff
@@ -0,0 +1,31 @@
+# Example toml file as input for metasyn
+
+[general]
+n_rows = 100
+
+
+[[var]]
+
```

Comment:
```suggestion
```

Reply:
I actually wanted the vars to be a little spaced away from the general table, do you disagree?

Reply:
nah, fine